lib/src/components/NxSidebarNavigation/NxSidebarNavitation.scss

```diff
@@ -0,0 +1,184 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../scss-shared/nx-variables';
+@import '../../scss-shared/nx-text-helpers';
+@import '../../scss-shared/nx-container-helpers';
+
+$nx-sidebar-width__open: 250px;
+$nx-sidebar-width__closed: 69px;
+$nx-sidebar-logo-height: 33px;
+$nx-sidebar-nav-background-color: $nx-blue-1000;
+$nx-sidebar-nav-text-color: $nx-white;
+$nx-sidebar-nav-help-color: $nx-grey-50;
+$nx-sidebar-nav-hover-color: $nx-indigo-600;
+$nx-sidebar-nav-selected-color: $nx-indigo-800;
+$nx-sidebar-separator-color: $nx-grey-900;
+
+
+.nx-sidebar-nav {
+  @include container-vertical;
+
+  box-sizing: border-box;
+  background-color: $nx-sidebar-nav-background-color;
+  color: $nx-sidebar-nav-text-color;
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+  min-width: 250px;
```

Comment:
You have a variable for this which isn't getting used here.  Additionally, is this really supposed to be a min or should it just be `width`?

Reply:
> You have a variable for this which isn't getting used here

Changed to use variable in fb8c6551007cf0bd70c9d0bd0b2437f1544155cf

> is this really supposed to be a min or should it just be `width`?

Maybe you could help me understand something here: 
if I use `width` the sidebar's width ends up being _less_ than 250px. If I use `min-width` the sidebar's width is _exactly_ 250px.

Reply:
> if I use width the sidebar's width ends up being less than 250px; If I use min-width the sidebar's width is exactly 250px. I don't understand why.

The sidebar probably needs `flex: none`.  `flex-shrink` defaults to 1, not 0, which I always find counter-intuitive

Reply:
> > if I use width the sidebar's width ends up being less than 250px; If I use min-width the sidebar's width is exactly 250px. I don't understand why.
> 
> The sidebar probably needs `flex: none`. `flex-shrink` defaults to 1, not 0, which I always find counter-intuitive

Fixed in 6c27955c7c30914aa6bcc37bd14edd4a4c8ddc65